src/Microsoft.ServiceFabric.Actors/Runtime/ActorBase.cs

```diff
@@ -99,7 +102,7 @@ public ActorService ActorService
 
         internal bool IsInitialized { get; set; }
 
-        internal DiagnosticsManagerActorContext DiagnosticsContext
+        internal DiagnosticActorContext DiagnosticsContext
```

Comment:
Context is a filler word. Since we're changing the property type, I would consider dropping it and naming the property simply Diagnostics.